src/app.js

```diff
@@ -1 +1,55 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+console.log('Hallo daar!');
+
+
+let countries;
+
+    async function fetchCountries() {
+        try {
+            const response = await axios.get('https://restcountries.com/v2/all?fields=name,region,flag,population,topLevelDomain');
+            countries = response.data
+            console.log(countries)
+            sortByPopulation(countries);
+            arrayToHtml(countries);
+        } catch (e) {
+            console.error(e);
+        }
+
+    }
+
+    function sortByPopulation() {
+        countries.sort((a, b) => {
+            return a.population - b.population
+        });
+    }
+
+    function findRegion(Array) {
+        return Array.region;
+
+    }
+
+    function arrayToHtml(Array) {
+
+        const countriesList = document.getElementById('Countries')
+
+
+        const countryMap = Array.map((country) => {
+
```

Comment:
Hier zou je nog spreading kunnen toepassen, zodat je niet overal `country.` vóór hoeft te zetten:
```suggestion
        const countryMap = Array.map(country => {
            const { name, flag, population } = country;
```